src/elements/image/ImageElementForm.tsx

```diff
@@ -53,34 +55,64 @@ const htmlLength = (text: string) => {
   return el.innerText.length;
 };
 
+export const thumbnailOption = {
+  text: "thumbnail",
+  value: "thumbnail",
+};
+
 export const ImageElementForm: React.FunctionComponent<Props> = ({
   errors,
   fields,
   fieldValues,
+  roleOptionsStore: RoleOptionsStore,
 }) => {
   return (
     <div data-cy={ImageElementTestId}>
       <Columns>
         <Column width={2 / 5}>
           <FieldLayoutVertical>
-            <CustomDropdownView
-              field={fields.role}
-              label="Weighting"
-              errors={errors.role}
-              options={
-                minAssetValidation(fieldValues.mainImage, "").length
+            <RoleOptionsStore>
+              {(additionalRoleOptions) => {
+                const roleOptions = minAssetValidation(
+                  fieldValues.mainImage,
```

Comment:
Still on the fence about this being a separate component. It might look a bit tidier if it was split out, but I acknowledge the number of props it would require.